src/components/shared/AdvantageCard.vue

```diff
@@ -0,0 +1,42 @@
+
+<script setup>
+const props=defineProps({
+    icon: {
+        type: Object,
+        required: true
+    },
+    name:{
+        type: String,
+        default: ''
+    },
+    description: {
+        type: String,
+        default: ''
+    },
+})
+</script>
+<template>
+    <div class="AdvantageCard">
```

Comment:
the class is not BEM, we don't use PascalCase for classes, try advante-section__card along with advantage-card. All BEM tree in this component needs to be patched.